lib/philomena/limits.ex

```diff
@@ -0,0 +1,53 @@
+defmodule Philomena.Limits do
+  @moduledoc """
+  Global limits for users.
+  """
+
+  #
+  # Tags changed per 10 minutes (50)
+  #
+
+  defp tag_count_key_for_ip(ip) do
+    "rltcn:#{to_string(ip)}"
+  end
+
+  def is_limited_for_tag_count?(nil, ip) do
+    amt = Redix.command!(:redix, ["GET", tag_count_key_for_ip(ip)]) || 0
+    amt > 50
```

Comment:
same as with my ratings comment, the rate limit here is 51 per 10 minutes, not 50, should be `>= 50` or `> 49`